src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateTFMRule.cs

```diff
@@ -75,20 +75,9 @@ private static string GetDisplayVersion(Version version)
             return sb.ToString();
         }
 
-        // TODO: When we have this inferred in the sdk targets, we won't need this
-        private AddPropertyTransform<NuGetFramework> OutputPathTransform =>
-            new AddPropertyTransform<NuGetFramework>("OutputPath",
-                f => $"bin/$(Configuration)/{f.GetShortFolderName()}",
-                f => true);
-
-        private AddPropertyTransform<NuGetFramework> FrameworkIdentifierTransform =>
-            new AddPropertyTransform<NuGetFramework>("TargetFrameworkIdentifier",
-                f => f.Framework,
-                f => true);
-
-        private AddPropertyTransform<NuGetFramework> FrameworkVersionTransform =>
-            new AddPropertyTransform<NuGetFramework>("TargetFrameworkVersion",
-                f => "v" + GetDisplayVersion(f.Version),
-                f => true);
+        private AddPropertyTransform<IEnumerable<NuGetFramework>> FrameworksTransform =>
+            new AddPropertyTransform<IEnumerable<NuGetFramework>>("TargetFrameworks",
+                frameworks => string.Join(";", frameworks.Select(f => f.GetShortFolderName())),
```

Comment:
Can you give an example of one of these frameworks?

Given that the migrator is currently scoped to well-formed Preview2 projects which are generally netstandard/netcoreapp is this still a concern?